elements/pfe-navigation-account/src/pfe-navigation-account.js

```diff
@@ -83,6 +87,184 @@ class PfeNavigationAccount extends PFElement {
     this._userData = null;
     this._avatars = [];
 
+    // set default language, overridden by check in connected callback
+    this._lang = "en";
+
+    // Translations
+    this._navTranslations = {
```

Comment:
I'd like to talk more as a team about alternate ways to capture content besides embedding it in components.  This makes the component very difficult to reuse in other situations and scopes it specifically for Red Hat-only.  Perhaps passing the data through an API or adding support for a JSON object is a way to go?  Kendall and I have gone down this path in a few different ways and would be happy to consult.  In the past, we've had a lot of consensus around not putting content into a component. cc @starryeyez024

Reply:
Sounds awesome to me, I think this is good enough for the time being. Definitely not the long term solution though.